config/dockerfiles/Dockerfile.Rust

```diff
@@ -0,0 +1,27 @@
+FROM debian:jessie
```

Comment:
FYI: This image is based off of https://hub.docker.com/r/jimmycuadra/rust/

But as I don't know Jimmy Cuadra from Adam I don't feel the necessary level of trust to just point to his Docker image.  Future updates could be of compromised security.

/cc @bokmann Do we have a Docker image policy?